tools/azsdk-cli/Azure.Sdk.Tools.Cli.Contract/Azure.Sdk.Tools.Cli.Contract.csproj

```diff
@@ -1,19 +1,22 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
-    <!-- Brings in Microsoft.Extensions.* + Microsoft.AspNetCore.* abstractions -->
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
I'm a bit concerned about the removal of the Framework ASp.NET core ref, but if it all compiles and tests /shrug

Reply:
Ah not on this one though. NM. This is just contract it's removed from.

Reply:
This may have been a bad rebase then